app/frontend/pairings/PlayerName.svelte

```diff
@@ -12,7 +12,11 @@
         {player.side_label}
     {/if}
     <div class="ids" style="{$showIdentities ? 'display: block;' : ''}">
-        <Identity identity={player.corp_id}/>
-        <Identity identity={player.runner_id}/>
+        {#if player.side_label}
+            <Identity identity={player.side_label == '(Corp)' ? player.corp_id : player.runner_id }/>
```

Comment:
This could be based on the side enum rather than a label.

Reply:
i'm going to leave that for a future refactoring.

Reply:
Sure thing.  Done.